meson.build

```diff
@@ -205,7 +203,7 @@ executable(
                   libhugetlbfs_dep ],
   include_directories: incdir,
   install: true,
-  install_dir: get_option('sbindir')
+  install_dir: join_paths(prefixdir, get_option('sbindir'))
```

Comment:
For consistency sake I suggest you introduce a new variable and apply the prefix as we have for mandir, bindir etc.

Reply:
addressed :)